deployment/ccip/changeset/cs_jobspec.go

```diff
@@ -25,31 +26,40 @@ func CCIPCapabilityJobspec(env deployment.Environment, _ any) (deployment.Change
 	// Find the bootstrap nodes
 	nodesToJobSpecs := make(map[string][]string)
 	for _, node := range nodes {
+		// pick first keybundle of each type (by default nodes will auto create one key of each type for each defined chain family)
+		keyBundles := map[string]string{}
+		for details, config := range node.SelToOCRConfig {
+			family, err := chainsel.GetSelectorFamily(details.ChainSelector)
+			if err != nil {
+				break
+			}
+			_, exists := keyBundles[family]
+			if !exists {
```

Comment:
The previous code had comment "// TODO: Validate that that all EVM chains are using the same keybundle."

Based on that, does it make sense to validate that if a bundle exists for a family, ensure it is same

keyBundleId, exists := keyBundles[family]
if exists {
	if keyBundleId != config.KeyBundleID
		return deployment.ChangesetOutput{}, new Error("A family is not using the same keyBundle")
}

Reply:
I resolved the TODO since that's the implicit behaviour we've used for keystone too. A single key is automatically generated for each family (as long as there's a relayer for that family type), so using different ones is unexpected

I'll add that check in 👍🏻